cmd/flags.go

```diff
@@ -47,6 +48,14 @@ func gRPCFlags(cmd *cobra.Command, v *viper.Viper) *cobra.Command {
 	return cmd
 }
 
+func MemoFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().String(flagMemo, "", "a memo to include in transaction")
+	if err := v.BindPFlag(flagMemo, cmd.Flags().Lookup(flagMemo)); err != nil {
+		panic(err)
+	}
+	return cmd
+}
+
```

Comment:
Small nit but since this function is only used in the `cmd` package it doesn't need to be exported.